.gitignore

```diff
@@ -3,3 +3,4 @@ elm-stuff
 # elm-repl generated files
 repl-temp-*
 node_modules
+dist
```

Comment:
nice! You should not need to build - so you can also delete that dir if you want.

Reply:
Ed helped me with that as I was getting some errors before

Reply:
:) Teamwork!